lms/templates/courseware/courseware-chromeless.html

```diff
@@ -134,17 +134,21 @@
 
         if (eventType !== 'load' && newWidth === lastWidth && newHeight === lastHeight) {
             // Monitor when any anchor tag is clicked, it is checked to make sure
-            // it is referencing an element's id (not an external website). If
-            // the href attribute is an id, the location of the selected focus
+            // it is referencing an element's id or name (not an external website). If
+            // the href attribute is an id or name, the location of the selected focus
             // element is sent through its offset attribute. The offset will
             // allow the page to scroll to the location of the focus element so
-            // that it is at the top of the page.
+            // that it is at the top of the page. Unique ids and names are
+            // required for proper scrolling.
 
             $('a').on("click", function(event){
               if ($(this).attr('href')[0] === "#") {
-                event.preventDefault()
```

Comment:
Question: How did this JS code ever work without trailing semicolons?

Reply:
🤷‍♀️